### PR TITLE
moving eslint to extension

### PR DESCRIPTION
**Pull request by ZakMiller**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -50,7 +39,7 @@ function build() {
     .pipe(gulp.dest('public'))
 }
 
-function watch() {
+function watch () {
```
Comment (SylvanSign) [inline gulpfile.js]: I'm not a fan of the space before function parens. Are you?

Comment (SylvanSign) [reply gulpfile.js]: I'd prefer to add this to the `.eslintrc.js`:
```json

  "rules": {
    "space-before-function-paren": ["error", "never"]
  }

**Review by SylvanSign — CHANGES_REQUESTED** [review]
.

File: gulpfile.js (hunk 1 above)
Comment (ZakMiller) [reply gulpfile.js]: Hmm... it's detecting it as an error now, but it's not autofixing. Any idea why?

Comment (SylvanSign) [reply gulpfile.js]: Yeah. You actually have to change the file in order for the linter to run on save, it seems. Just add a space somewhere, then save.

File: README.md [hunk 2]
```diff
@@ -14,6 +14,21 @@
     $ npm install -g gulp
   ```
 
+4. Get Visual Studio Code Extensions ESLint and beautify.
+
+
+5. Add these to the VSCode settings.json file:
+``` sh
```
Comment (SylvanSign) [inline README.md]: sh means shell. You want json I think

**Review by SylvanSign — CHANGES_REQUESTED** [review]

**Review by SylvanSign — APPROVED** [review]

**Pull request merged**